services/web-app/components/PageNotFound/PageNotFound.js

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
We follow kebab case for component file names. I'm pretty sure we made this decision based on Next.js conventions. Can you update?